<ui version="4.0" >
 <class>Kontrahenci</class>
 <widget class="QDialog" name="Kontrahenci" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dodaj kontrahenta</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>Nazwa:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel3" >
         <property name="text" >
          <string>Miejscowość:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel4" >
         <property name="text" >
          <string>Kod:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel2" >
         <property name="text" >
          <string>Adres:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel2_2" >
         <property name="text" >
          <string>NIP:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel5" >
         <property name="text" >
          <string>Nr konta:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel6" >
         <property name="text" >
          <string>Typ:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLineEdit" name="nameEdit" />
       </item>
       <item>
        <widget class="QLineEdit" name="placeEdit" />
       </item>
       <item>
        <widget class="QLineEdit" name="codeEdit" />
       </item>
       <item>
        <widget class="QLineEdit" name="addressEdit" />
       </item>
       <item>
        <widget class="QLineEdit" name="nipEdit" >
         <property name="inputMask" >
          <string>999-99-999-99; </string>
         </property>
         <property name="text" >
          <string>---</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="accountEdit" >
         <property name="inputMask" >
          <string>99-9999-9999-9999-9999-9999-9999; </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeCombo" >
         <item>
          <property name="text" >
           <string>Firma</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Urząd</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="textLabel5_2" >
         <property name="text" >
          <string>Telefon:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel5_2_2" >
         <property name="text" >
          <string>e-mail</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel5_2_2_2" >
         <property name="text" >
          <string>www:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLineEdit" name="telefonEdit" />
       </item>
       <item>
        <widget class="QLineEdit" name="emailEdit" />
       </item>
       <item>
        <widget class="QLineEdit" name="wwwEdit" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>10</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" />
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>O&amp;K</string>
       </property>
       <property name="shortcut" >
        <string>Alt+K</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Can&amp;cel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Kontrahenci</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
